packages/slice-machine/pages/slices.tsx

```diff
@@ -145,21 +167,64 @@ const SlicesIndex = ({ env }: { env: Environment }) => {
                 </Box>
               )}
             </Box>
-          ) : null}
-          <Box>
-            {libraries &&
-              libraries.map((maybelib: LibraryState | undefined, i) => {
+          )}
+          <Box
```

Comment:
Once again it's not mandatory, but I think it should be a best practice to externalise those components